packages/playground/cli/tests/run-cli.spec.ts

```diff
@@ -18,229 +18,92 @@ import {
-			expect(response.httpStatusCode).toBe(200);
-		});
-
-		test('should run a static html project using --auto-mount', async () => {
-			vi.spyOn(process, 'cwd').mockReturnValue(
-				path.join(import.meta.dirname, 'mount-examples', 'static-html')
-			);
+		test('should set WordPress version', async () => {
+			const oldestSupportedVersion =
+				MinifiedWordPressVersionsList[
+					MinifiedWordPressVersionsList.length - 1
+				];
 			cliServer = await runCLI({
+				...suiteCliArgs,
 				command: 'server',
-				autoMount: true,
+				wp: oldestSupportedVersion,
 			});
-			const response = await cliServer.playground.request({
-				url: '/',
-				method: 'GET',
-			});
-			expect(response.httpStatusCode).toBe(200);
-			expect(response.text).toContain('<title>Static HTML</title>');
-		});
-
-		test('should run a php project using --auto-mount', async () => {
-			vi.spyOn(process, 'cwd').mockReturnValue(
-				path.join(import.meta.dirname, 'mount-examples', 'php')
+			await cliServer.playground.writeFile(
+				'/wordpress/version.php',
+				`<?php
+            require_once '/wordpress/wp-load.php';
+            echo get_bloginfo("version");
+            ?>`
 			);
-			cliServer = await runCLI({
-				command: 'server',
-				autoMount: true,
-			});
 			const response = await cliServer.playground.request({
-				url: '/',
+				url: '/version.php',
 				method: 'GET',
 			});
 			expect(response.httpStatusCode).toBe(200);
-			expect(response.text).toContain('Hello world');
+			expect(response.text).toContain(oldestSupportedVersion);
 		});
 
-		test('should run a wordpress project using --auto-mount', async () => {
-			const tmpDir = await mkdtemp(
-				path.join(tmpdir(), 'playground-test-')
-			);
-			vi.spyOn(process, 'cwd').mockReturnValue(
-				path.join(tmpDir, 'wordpress')
-			);
-
-			const zip = await fetch('https://wordpress.org/latest.zip');
-			const zipPath = path.join(tmpDir, 'wp.zip');
-			await writeFile(zipPath, new Uint8Array(await zip.arrayBuffer()));
-			await promisify(exec)(`unzip "${zipPath}" -d "${tmpDir}"`);
-
-			const checksum = await getDirectoryChecksum(tmpDir);
-
+		test('should run blueprint', async () => {
 			cliServer = await runCLI({
+				...suiteCliArgs,
 				command: 'server',
-				autoMount: true,
+				blueprint: {
+					steps: [
+						{
+							step: 'setSiteOptions',
+							options: {
+								blogname: 'My Blog Name',
+							},
+						},
+					],
+				},
 			});
 			const response = await cliServer.playground.request({
 				url: '/',
 				method: 'GET',
 			});
 			expect(response.httpStatusCode).toBe(200);
-			expect(response.text).toContain(
-				'<title>My WordPress Website</title>'
-			);
-
-			/**
-			 * Playground should not modify the mounted directory.
-			 */
-			expect(await getDirectoryChecksum(tmpDir)).toBe(checksum);
+			expect(response.text).toContain('<title>My Blog Name</title>');
 		});
 
 		test('should be able to follow external symlinks in primary and secondary PHP instances', async () => {
```

Comment:
This test was incorrectly included within the auto-mount tests, so this is just moving it where it belongs.